### ISSUE TITLE
Avoid double checkin in ContentResource when saving content

### ISSUE THREAD

**Issue by jgambarios** [dotCMS/core#15053]
Using the `com.dotcms.rest.ContentResource` we are able to save content and request the execution of a Workflow Action, example:

```
curl -v -u admin@dotcms.com:admin -XPUT http://localhost:8080/api/content/MyCustomAction/1 -H "Content-Type:application/json" -d '{
    stName:"webPageContent",
    languageId:1,
    title:"Test1",
    contentHost:"demo.dotcms.com",
    body:"Test body1"
}'
```

But if, and using the example above, `MyCustomAction` has a save operation (for example a Save actionlet) we will be saving the contentlet twice, that's not necessary as we can now know if a Workflow Action is saving content.

## Expected Behavior
Don't save twice the contentlet

## Possible Solution
If the Workflow Action has a save operation we could fire directly the Workflow action instead of execute the regular `checkin` method.

**Comment by jgambarios** [comment]
PR: https://github.com/dotCMS/core/pull/15127

**Comment by bryanboza** [comment]
Fixed, new enhancement in the #15317
